codec_test.go

```diff
@@ -222,6 +222,17 @@ func TestMarshalText(t *testing.T) {
 	}
 }
 
+func TestDecodePlainWithWrongLength(t *testing.T) {
+	arg := []byte{'4', '2'}
+
+	u := UUID{}
+	err := u.decodePlain(arg)
```

Comment:
@zerkms I might be nitpicking here a bit. Since we do not use the error value, how do we feel about simplifying this to the single if expression?

```Go
if u.decodePlain(arg) == nil {}
```

Reply:
The alternative I'd go with, if we care about the error type, would be:

```Go
var err error
// ...
if err = u.decodePlain(arg); err == nil {}
```

Just so that we assert the return type as being an `error` value.

Reply:
I don't mind in either, but would personally prefer the first (given every other test case does not care about the exact returned `err` type either).